compiler/testData/ir/irText/expressions/destructuring1.kt.txt

```diff
@@ -31,3 +31,4 @@ fun B.test() {
   val y: Int = (<this>, tmp0_container).component2()
   // }
 }
+
```

Comment:
Maybe such changes in irText tests should be reverted

Reply:
(un)done!